google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/testing/LocalPubSubHelperTest.java

```diff
@@ -50,25 +46,10 @@ public void testCreate() {
   }
 
   @Test
-  public void testOptions() {
-    LocalPubSubHelper helper = LocalPubSubHelper.create();
-    PubSubOptions options = helper.getOptions();
-    assertTrue(options.getProjectId().startsWith(PROJECT_ID_PREFIX));
-    assertTrue(options.getHost().startsWith("localhost:"));
-    assertSame(NoCredentials.getInstance(), options.getCredentials());
-  }
-
-  @Test
   public void testStartStopReset() throws IOException, InterruptedException, TimeoutException {
     LocalPubSubHelper helper = LocalPubSubHelper.create();
     helper.start();
-    PubSub pubsub = helper.getOptions().getService();
-    pubsub.create(TopicInfo.of(TOPIC));
-    assertNotNull(pubsub.getTopic(TOPIC));
```

Comment:
I think this buys us very little. This method is essentially testing two things:
- start, reset, stop
- getting service from the helper

We're still doing the first, but now that PubSub and PubSubOptions are gone, it's tough to do the second. Maybe we can provide a way to get PublisherClient and SubscriberClient from the helper? If we want to do it, I think that should be in a separate PR. Note that it still won't be usefully testing the clients. The clients are already tested with the generated unit and smoke tests. The test is only useful in verifying that the clients actually communicate with the created helper, as opposed to the real service.